msal-client-credentials-flow/app.js

```diff
@@ -48,6 +48,10 @@ app.get('/api/application', (req, res) => {
       headers: { Authorization: `Bearer ${response.accessToken}` }
     }
 
+    // MSAL Node uses an in-memory token cache by default
+    // Repeated HTTP GET requests will utilize the cached token until it nears expiration, at which point a new access token will be requested.
+    console.log(`\n\nToken: ${response.accessToken}`)
+
```

Comment:
Wouldn't it be better to log based on `response.fromCache`?  No need to dump the whole thing.  You can build a log statement that says it if was from cache or newly acquired.

Reply:
Absolutely, great idea